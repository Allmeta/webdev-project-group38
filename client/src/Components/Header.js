import React, { Component } from 'react'
import styled from 'styled-components'
import Logo from '../images/filmguru-logo2.jpg'
import SearchForm from './SearchForm.js'

// The logo uses this hex-color: #4553F4

const HDiv = styled.div`
  position:relative;
  padding: 0;
  padding-left:10%;
  padding-right:10%;
  margin: 0;
  background-color: #4553F4;
  font-family: Verdana, Geneva, Tahoma, sans-serif;
  

  @media screen and (max-width: 500px) {
    flex-direction: column;
  }
`
const ImgStyle = styled.img`
  width:200px;
  float: left;
`
const SearchFields = styled.div`
  float:left;
  position:absolute;
  width:1000px;
  left:40%;
  top:30px;
`

class Header extends Component {
  render () {
    return (
      <HDiv>
        <ImgStyle src={Logo} />
        <SearchFields>
          <SearchForm />
        </SearchFields>
      </HDiv>
    )
  }
}
export default Header
