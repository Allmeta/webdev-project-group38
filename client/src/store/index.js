import { createStore, applyMiddleware, compose } from 'redux'
import { SearchFormReducer } from "../reducers/SearchFormReducer";
import thunk from 'redux-thunk'

const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const store = createStore(SearchFormReducer, composeEnhancers(
  applyMiddleware(thunk)
));

export default store;