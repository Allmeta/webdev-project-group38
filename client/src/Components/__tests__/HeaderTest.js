import React from 'react'
import Header from '../Header.js'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import store from '../../store/index.js'
import { Provider } from 'react-redux'

it('renders correctly', () => {
  const tree = renderer.create(<Provider store={store}><Header /></Provider>).toJSON()
  expect(tree).toMatchSnapshot()
})
