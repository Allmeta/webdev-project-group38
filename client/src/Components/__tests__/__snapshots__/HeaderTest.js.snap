// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="sc-bdVaJa keoVwn"
  >
    MovieMate
    <p>
      Search/sort components go here OR: 
    </p>
  </div>
</div>
`;
